server/svix-server/src/v1/utils/mod.rs

```diff
@@ -199,6 +227,7 @@ fn list_response_inner<T: ModelOut>(
     limit: usize,
     is_prev_iter: bool,
```

Comment:
Ooooof, three `bool` params in a row is really to misuse.

I guess it's fine since the function is private, and the callers are right below. I'm wondering if there's a way to make the params clearer.